src/11_args.py

```diff
@@ -21,13 +29,15 @@
 a = [7, 6, 5, 4]
 
 # What thing do you have to add to make this work?
-print(f2(a))    # Should print 22
+print(f2(sum(a)))    # Should print 22
```

Comment:
Can't argue with this solution! What they do want is `f2(*a)` which basically unpacks that list into a usable form for the function